src/components/NcActions/NcActions.vue

```diff
@@ -807,6 +807,119 @@ p {
 }
 </style>
 ```
+
+## NcActions children limitations
+
+`<NcActions>` is supposed to be used with direct `<NcAction*>` children.
+Thought, it will work when actions are not direct children but are wrapped into custom components, it has limitations:
+- No `inline` prop property, including a single action display;
+- Accessibility issues, including changed keyboard behavior;
+- Invalid HTML.
+
+```
+<template>
+	<table class="actions-limitations-table">
+		<tr>
+			<th style="width: 50%">Non-direct children</th>
+			<th style="width: 50%">Direct NcAction* children</th>
+		</tr>
+
+		<tr>
+			<th colspan="2">This single button is supposed to be rendered as inline action but it is rendered as a menu:</th>
+		</tr>
+		<tr>
+			<td>
+				<NcActions>
+					<MyUserActionButton />
+				</NcActions>
+			</td>
+			<td>
+				<NcActions>
+					<NcActionButton icon="icon-user-white">Button</NcActionButton>
```

Comment:
I know this is just an example for the docs, but we don't want to use the `icon` prop anymore 🙈

Reply:
Yeah, but I wanted an example to be smaller. We haven't marked them as deprecated though.

Reply:
is `icon-user-white` a good idea to use? it's not visible with a light theme
![image](https://github.com/nextcloud-libraries/nextcloud-vue/assets/93392545/b4c14338-c728-455b-9e51-2497b3bff52f)

Reply:
`--icon-use-[white/dark]` is defined on the user-preferred scheme. They require `data-theme` attrs on styleguidist page.

Moving to out preferred SVG in `#icon` slot then.

Template becomes huge, added `v-for`.

![image](https://github.com/nextcloud-libraries/nextcloud-vue/assets/25978914/47bedba7-23ee-427e-aeaa-10b9aaa24f04)